app/controllers/api/base_controller.rb

```diff
@@ -39,8 +39,8 @@ class BaseController < ActionController::Base
 
     rescue_from Foreman::MaintenanceException, :with => :service_unavailable
 
-    def get_resource
-      instance_variable_get(:"@#{resource_name}") || raise('no resource loaded')
+    def get_resource(options = {})
```

Comment:
Debian Jessie has ruby 2.1, Ubuntu Trusty seems to have 2.0 only but maybe it's time to drop Trusty support in 1.15, @mmoll what do you think? I'd like to start using keyword arguments :-)